src/develop/ui/patterns/mobile/carousel.md

```diff
@@ -5,46 +5,42 @@ summary:
 
 # Carousel Pattern
 
-The Carousel Pattern
-enables you to display multiple items in a horizontal slide. Elements inside
-the placeholder will be split into different items. For dynamic content, use a
-List directly inside the placeholder. The Carousel block is ideal to create
-horizontal slides in smaller screens.
-
-Here's the preview in Service Studio:
+You can use the Carousel UI Pattern to display multiple items in a horizontal slide. This pattern is ideal for creating horizontal slides in smaller screens. You can also use this pattern for dynamic content, by placing a List directly inside the placeholder. 
 
 ![](images/Carousel_studio_preview.png)
 
-## How to Use the Carousel Pattern
+## How to use the Carousel UI Pattern
+
+1. In Service Studio, in the Toolbox, search for `Carousel`.
+  
+     The Carousel widget is displayed.
+
+    ![](images/carousel-widget.png)
+  
+
+1. From the Toolbox, drag the Carousel widget onto your application's screen. 
 
-Drag the elements into the placeholder to split them into different items. For
-dynamic content, use a List directly inside the placeholder, disable
-virtualization and set the **Animated** items to _False_ . The List Items will
-be Carousel Items, by default.
+    ![](images/Carousel_drag_pattern.png)
 
-1\. Drag the Carousel Pattern into your screen.
+1. Place your content into the **Carousel Items** placeholder. 
 
-![](images/Carousel_drag_pattern.png)
+    To use a List, drag it into **Carousel Items** placeholder (disable virtualization and set **Animated items** to False .
 
-2\. Place your content into the **CarouselItems** placeholder. To use a List,
-drag it into **CarouseItems** placeholder (disable virtualization and set
-**Animated items** to _False_ ).
+    ![](images/Carousel_items.png)
 
-![](images/Carousel_items.png)
 
-3\. All available options have default parameters, but you can change them.
+1. All available options have default properties, but you can change them.
```

Comment:
"All available properties have default values, but you can control the look and feel of the Carousel by changing them".